en/services/parameter.md

```diff
@@ -65,11 +65,16 @@ This key->value pair has many important properties:
 * Support for unknown autopilots, as long as they implement the protocol, is guaranteed
 * Adding a parameter is only a change to the onboard code.
 
-### Read Parameters
+### Read All Parameters {#read_all}
+
+A GCS should read all parameters when first connecting to a system (it does not contain a list of the vehicle parameters on startup).
+
+> **Note** The Drone will emit a [PARAM_VALUE](../messages/common.md#PARAM_VALUE) whenever a parameter is [written/changed](#write).
+  Provided the GCS keeps track of changed parameters, it should not need to re-read the full parameter list.
 
 Reading the parameter list is activated by sending the [PARAM_REQUEST_LIST](../messages/common.md#PARAM_REQUEST_LIST) message. 
-The onboard component should start to transmit the parameters individually after receiving this message. 
-The sending should be delayed after each parameter, in order to not use up the full radio bandwidth.
+The target component should start to broadcast the parameters individually in `PARAM_VALUE` messages after receiving this message. 
+The drone should allow a pause after sending each parameter in order to not use up the full radio bandwidth.
```

Comment:
HOw big a pause? Can we provide guidance here? How big is the gap for PX4?

Reply:
Depends on the bandwidth. Wild guess, do writes at 10 Hz max? It is not often that you have to set more than 10 params at once anyway.

Reply:
@julianoes So this is when you first connect and you're getting all the params at once - definitely more than 10 params. Does that change your answer?

Reply:
Oh, right, I got that wrong. Yes faster but depending on the bandwidth really. You could say up to 30-50% of the link bandwidth?